tests/integration/test_decompositions_integration.py

```diff
@@ -555,7 +679,6 @@ def test_Pgate_decomp_equal(self, setup_eng, s, tol):
         eng.run(prog)
         assert np.all(eng.backend.is_vacuum(tol))
 
-    @pytest.mark.backends("fock","tf","gaussian")
```

Comment:
Are these removed since they run on _all_ backends?

Reply:
Yes! The conftest file defaults to run tests on all backends.